tests/Doctrine/Tests/ORM/Functional/Ticket/GH11128Test.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Query;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+class GH11128Test extends OrmFunctionalTestCase
```

Comment:
Instead of adding a new test-case and model, you can just add another test to the existing by adapting for example `EagerFetchCollectionTest::testSubselectFetchJoinWithNotAllowed`